src/mmrelay/matrix/sync_bootstrap.py

```
@@ -0,0 +1,1201 @@
+NIO_COMM_EXCEPTIONS = facade.NIO_COMM_EXCEPTIONS
+JSONSCHEMA_VALIDATION_ERROR = facade.JSONSCHEMA_VALIDATION_ERROR
+SYNC_RETRY_EXCEPTIONS = facade.SYNC_RETRY_EXCEPTIONS
+WHOAMI_USER_ID_FALLBACK_EXCEPTIONS = facade.WHOAMI_USER_ID_FALLBACK_EXCEPTIONS
+
+__all__ = [
+    "_perform_initial_sync",
+    "_post_sync_setup",
+    "connect_matrix",
+    "login_matrix_bot",
+    "join_matrix_room",
+]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)